scripts/moneat-agent/moneat_agent/checks.py

```
@@ -0,0 +1,86 @@
+            log.success(f"CodeRabbit check completed after {int(elapsed)}s")
+            return True
+        else:
+            log.system(f"  Poll #{attempt}: CodeRabbit status = {status} ({int(elapsed)}s elapsed)")
+
+        time.sleep(poll_interval)
+
+
+def _get_coderabbit_status(repo: str, pr_number: int) -> str | None:
+    """Query check runs for the PR and return the CodeRabbit status, or None."""
+    try:
+        raw = subprocess.run(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)